docs/tutorials/intro-to-rust.md

```diff
@@ -0,0 +1,574 @@
+
+The example shown here will be a simple smart contract that serves as a counter, incrementing, decrementing, and returning the counter value. There is no previous Rust development required for this example. Folks familiar with the language may choose to jump straight into the examples below. 
+
+**Note**: these examples are meant to be run by cloning the parent repository and following the directions in their corresponding `README.md` in the example's directory.
+
+- [Cross-contract functions, like promises, or money transfers](https://github.com/nearprotocol/near-bindgen/tree/master/examples/cross-contract-high-level)
+- [Cross-contract calls using more low-level promise calls](https://github.com/nearprotocol/near-bindgen/tree/master/examples/cross-contract-low-level)
+- [Fungible token like ERC-20 but designed for the async runtime like NEAR](https://github.com/nearprotocol/near-bindgen/tree/master/examples/fun-token)
+- [Implements simulation of a distributed network of drones interacting with the mission control system](https://github.com/nearprotocol/near-bindgen/tree/master/examples/mission-control)
+- [Records the status messages of the accounts that call this contract](https://github.com/nearprotocol/near-bindgen/tree/master/examples/status-message-collections)
+- [Records the status messages of the accounts that call this contract](https://github.com/nearprotocol/near-bindgen/tree/master/examples/status-message)
+
+For those who won't wish to dive into the deep end, consider this page a safe "wading pool" with no diving signs posted.
+
+## 2-Step Rust Installation
+
+### 1. Install Rustup
+
+`curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh`
+
+([Taken from official installation guide](https://www.rust-lang.org/tools/install))
+
+### 2. Add wasm target to your toolchain
+
+`rustup target add wasm32-unknown-unknown`
+
+<blockquote class="info">
+  <a href="https://doc.rust-lang.org/edition-guide/rust-2018/platform-and-target-support/webassembly-support.html" target="_blank">Why <code>unknown-unknown</code>?</a>
+</blockquote>
+
+## Resources
+
+The online book [from the official Rust site](https://doc.rust-lang.org/1.30.0/book/2018-edition/ch00-00-introduction.html) is a great resource to start with. It's recommended to read Chapter 1 before continuing, especially the `cargo` section.
+
+This book is also available with [Dash](https://kapeli.com/dash) for offline usage in an easy-to-read interface. Once Dash is installed, it can be found within the app or at the [docset Github page](https://github.com/indirect/dash-rust). 
```

Comment:
this looks really outdated

![image](https://user-images.githubusercontent.com/24913/75221853-defb6000-57b3-11ea-9fd0-2a3cc879b804.png)

Reply:
good point, this is kind of overkill anyway, I'll just mentioned to get it in Dash. I had a voice in my head telling me this was too much. Yanking that part.